runtime/compiler/control/J9Recompilation.cpp

```diff
@@ -561,10 +561,9 @@ J9::Recompilation::getCounterSymRef()
    }
 
 
-/////////////////////////
+
 // DEBUG
 //////////////////////////
-
```

Comment:
Please add back this blank line that was removed.  It looks like `J9Recompilation.cpp` doesn't actually have any line of comments that are being incorrectly recognized by Doxygen.